source/renderer/app/containers/TopBarContainer.js

```diff
@@ -11,6 +11,7 @@ import menuIconOpened from '../assets/images/menu-opened-ic.inline.svg';
 import menuIconClosed from '../assets/images/menu-ic.inline.svg';
 import { matchRoute } from '../utils/routing';
 import { ROUTES } from '../routes-config';
+import { IS_STAKING_INFO_PAGE_AVAILABLE } from '../config/stakingConfig';
```

Comment:
@mchappell please introduce a `IS_TADA_ICON_AVAILABLE` configuration variable to be used for the icon checks 🙏